plugins/flutter_aepedgeidentity/README.md

```diff
@@ -1,6 +1,356 @@
+
+ > Note: It is required to initialize the SDK via native code inside your AppDelegate (iOS) and MainApplication class (Android).
+
+As part of the initialization code, make sure that you set the SDK wrapper type to `Flutter` before you start the SDK.
+
+Refer to the [Initialization](https://github.com/adobe/aepsdk_flutter#initializing) section of the root README for more information about initializing the SDK.
+
+**Initialization Example**
+
+iOS
+```objc
+// AppDelegate.h
+@import AEPCore;
+@import AEPEdge;
+@import AEPEdgeIdentity;
+...
+@implementation AppDelegate
+
+// AppDelegate.m
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    [AEPMobileCore setWrapperType:AEPWrapperTypeFlutter];
+
+     // TODO: Set up the preferred Environment File ID from your mobile property configured in Data Collection UI
+    NSString* ENVIRONMENT_FILE_ID = @'YOUR-APP-ID';
+    
+    const UIApplicationState appState = application.applicationState;
+
+    NSArray *extensionsToRegister = @[AEPMobileEdgeIdentity.class, 
+                                      AEPMobileEdge.class                                             
+                                      ];
+
+    [AEPMobileCore registerExtensions:extensionsToRegister completion:^{
+    [AEPMobileCore configureWithAppId: ENVIRONMENT_FILE_ID];
+    }];
+    return YES;   
+ } 
+```
+
+Android
+```java
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Edge;
+import com.adobe.marketing.mobile.edge.identity.Identity;  
+...
+import io.flutter.app.FlutterApplication;
+...
+public class MainApplication extends FlutterApplication {
+  ...
+  // TODO: Set up the preferred Environment File ID from your mobile property configured in Data Collection UI
+  private final String ENVIRONMENT_FILE_ID = 'YOUR-APP-ID';
+
+  @Override
+  public void on Create(){
+    super.onCreate();
+    ...
+    MobileCore.setApplication(this);
+    MobileCore.setWrapperType(WrapperType.FLUTTER);
+
+    Edge.registerExtension();
+    Identity.registerExtension();
+    MobileCore.start(new AdobeCallback () {
+        @Override
+        public void call(Object o) {
+          MobileCore.configureWithAppID(ENVIRONMENT_FILE_ID);
+        }
+   });
+```
+------
+### Importing the extension
+In your Flutter application, import the Edge Identity extension as follows:
+```dart
+import 'package:flutter_aepedgeidentity/flutter_aepedgeidentity.dart';
+import 'package:flutter_aepedgeidentity/flutter_aepedgeidentity_data.dart';
+```
+------
+## API reference
+### extensionVersion
+Returns the SDK version of the Identity for Edge Network extension
+
+**Syntax**
+```dart
+static Future<String> get extensionVersion
+ ```
+
+**Example**
+ ```dart
+String version = await Identity.extensionVersion;
+ ```
+------
+### getExperienceCloudId
+Returns the Experience Cloud ID. An empty string is returned if the Experience Cloud ID was previously cleared.
```

Comment:
for all apis can use the description from https://aep-sdks.gitbook.io/docs/foundation-extensions/identity-for-edge-network/api-reference#getexperiencecloudid instead